src/main/groovy/com/avioconsulting/mule/linter/rule/configuration/GlobalConfigRule.groovy

```diff
@@ -0,0 +1,48 @@
+package com.avioconsulting.mule.linter.rule.configuration
+
+import com.avioconsulting.mule.linter.model.Application
+import com.avioconsulting.mule.linter.model.ConfigurationFile
+import com.avioconsulting.mule.linter.model.MuleComponent
+import com.avioconsulting.mule.linter.model.Rule
+import com.avioconsulting.mule.linter.model.RuleViolation
+
+class GlobalConfigRule extends Rule {
+
+    static final String RULE_ID = 'GLOBAL_CONFIG'
+    static final String RULE_NAME = 'Global mule configuration xml exists and contain required configuration.'
+    static final String RULE_VIOLATION_MESSAGE = 'Mule configuration xml contain global configuration: '
+    static Map<String, String> DEFAULT_NONE_GLOBAL = ['sub-flow': 'http://www.mulesoft.org/schema/mule/core',
+                                                      'flow'    : 'http://www.mulesoft.org/schema/mule/core']
+    static Map<String, String> noneGlobalElements
+    static String DEFAULT_FILE_NAME = 'globals.xml'
+    String globalFileName
+
+    GlobalConfigRule(String globalFileName, Map<String, String> noneGlobalElements) {
```

Comment:
The array that contains flow and sub-flow shouldn't be overwritten, flow and sub-flow should never be considered globals, that shouldn't be a configuration.  Instead we should allow a user to configure additional elements (which probably are actually gobals) that they would like to ignore/allow in regular config files.  An example might be allowing apikit router configs in regular configs still.